CHANGELOG.md

```diff
@@ -200,8 +200,21 @@ Adding a new version? You'll need three changes:
   that prefer to mount a file over binding a Secret to an environment variable
   value. Only one of the two options can be used.
   [#4808](https://github.com/Kong/kubernetes-ingress-controller/pull/4808)
+- New `KongUpstreamPolicy` CRD superseding `KongIngress.Upstream` was introduced.
+  It allows overriding Kong Upstream settings generated for a specific `Service` used
+  in an `Ingress` or Gateway API `*Route`. A policy can be applied to a `Service` by
```

Comment:
```suggestion
  in an `Ingress` or Gateway API `Route`. A policy can be applied to a `Service` by
```
in gateway-api docs e.g. [here](https://gateway-api.sigs.k8s.io/geps/gep-709/?h=route) they don't use `*ROute` to indicate `HTTPRoute`, `TCPRoute`,  etc. They just write `Route`